### PR TITLE
Add Ulule

### PR DESCRIPTION
**Pull request by TJ619**
The EU crowdfunding website (like Kickstarter for US)

### CODE REVIEW

**Review by stephengroat — APPROVED** [review]
thanks for the pr @TJ619

File: _data/other.yml [hunk 1]
```diff
@@ -343,6 +343,13 @@ websites:
       tfa: Yes
       software: Yes
 
+    - name: Ulule
+      url: https://ulule.com/
```
Comment (Carlgo11) [inline _data/other.yml]: Please change the URL to `https://www.ulule.com/`

**Review by Carlgo11 — DISMISSED** [review]

File: _data/other.yml [hunk 2]
```diff
@@ -343,6 +343,13 @@ websites:
       tfa: Yes
       software: Yes
 
+    - name: Ulule
+      url: https://www.ulule.com/
+      twitter: ulule
+      facebook: ulule
```
Comment (jamcat22) [inline _data/other.yml]: Their Facebook page redirects to `ululeEN` for me. Does anyone else get a different result after going to [facebook.com/ulule](https://www.facebook.com/ulule)?

**Review by jamcat22 — CHANGES_REQUESTED** [review]

File: _data/other.yml (hunk 2 above)
Comment (stephengroat) [reply _data/other.yml]: it's doing that based on globalization techniques available for facebook pages. it's the best option to link against

Comment (Carlgo11) [reply _data/other.yml]: Well `ulule` seems to be in French so why not use `ululeEn`? Also, it doesn't redirect for me.

Comment (TJ619) [reply _data/other.yml]: For me (in FR) the ulule facebook didn't redirect me to the FR one, seem to be better for all.

Comment (stephengroat) [reply _data/other.yml]: i believe that `ulule` is the company's [Facebook root or default page](https://www.facebook.com/business/help/331800410323820)

Comment (Carlgo11) [reply _data/other.yml]: I'm not saying it redirects to another facebook page. Just that `ulule` seems to be a French version.

## ulule
ulule-screenshot
## ululeEn
ululeEn-screenshot

Comment (stephengroat) [reply _data/other.yml]: for me, `ulule` redirects to `ulueEn`

Comment (Carlgo11) [reply _data/other.yml]: @stephengroat are you signed in to Facebook?

**Comment by stephengroat** [comment]
i get redirects when signed in and when not

i am behind a pretty aggressive proxy though

File: _data/other.yml (hunk 2 above)
Comment (TJ619) [inline _data/other.yml]: Oh yes i just see.
Yes the **ulule** page is the French Page (cause created in France before expand for EU)
So yes, better for here to put **ululeEn** page

**Comment by TJ619** [comment]
Yes the ulule page is the French Page (cause created in France before expand for EU)
So yes, better for here to put ululeEn page for all.

**Comment by jamcat22** [comment]
I think in the context of how we currently handle things (forcing English websites unless they're a regional service such as a bank) we should probably force their English Facebook page as well. If we later implement options for other languages then we can figure something out.

Cross ref #2510.

**Review by jamcat22 — APPROVED** [review]

**Pull request merged**